tetrateci/create_istio_release.sh

```diff
@@ -16,6 +16,11 @@ else
 fi
 
 ## Set up release-builder
+
+# BOM is needed for generating bill of materials, required by Istio since 1.13, https://github.com/istio/release-builder/pull/893
+go install sigs.k8s.io/bom/cmd/bom@v0.2.2
+cp go/bin/bom /usr/local/bin/
```

Comment:
just curious, is copying needed? in my dev env, or other cases, where binaries get from `go get` is already in a dir, which is part of `PATH`.

Reply:
I am not sure as i dont have the access of github runner instance. but in my ubuntu based environment, go install download the binary in a folder which is not in $PATH variable.

root@fipsbuild:~/go/bin# pwd
/root/go/bin
root@fipsbuild:~/go/bin# which go
/usr/local/go/bin/go

This is the $PATH for github runner as seen from github actions logs.
++ export PATH=/usr/local/go/bin:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin